account_analytic_sequence/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Account Analytic Sequence
```

Comment:
Splitting readme as per https://github.com/OCA/maintainer-tools/tree/master/template/module/readme would be great!